CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - feat(remix): Switch to OTEL setup (#593)
 - feat(remix): Update `start` script for built-in Remix servers (#604)
 - deps: Bump glob to `10.4.2` (#599)
+- feat(nextjs): Refactor Refactor logic ([#607](https://github.com/getsentry/sentry-wizard/pull/607))
```

Comment:
Can't suggest this in the GH UI but:

Can you add the changelog entry to a new `## Unreleased` section? Since you opened this PR we already released a new version, so the entry would be in the wrong version.